specifyweb/frontend/js_src/lib/localization/merging.ts

```diff
@@ -254,4 +254,13 @@ export const mergingText = createDictionary({
     'ru-ru': 'Связанные записи',
     'uk-ua': "Пов'язані записи",
   },
+  localityMerging: {
+    'en-us': 'Locality merging',
+  },
+  paleoContextMerging: {
+    'en-us': 'Paleo context merging',
+  },
+  collectingEventMerging: {
+    'en-us': 'Collecting event merging',
```

Comment:
Shouldn't these names be localized based on the table name assigned by the user in the schema? If possible, I think we should make it generalized so that it will be "`{Table Caption}` Merging" rather than  creating specific localization strings for each one.

Reply:
yes this was just for a test :) 
I'll keep this comment open to not gorget to change that